### ISSUE TITLE
auro-icon: Resize Icon When Label Exists

### ISSUE THREAD

**Issue by rallyb13** [AlaskaAirlines/auro-icon#89]
### Is your feature request related to a problem? Please describe.

The `customSize` attribute works great to resize, but only when there's no label.
Because it targets the parent element rather than the icon itself, any resize attempt ends up targeting the label as well. Which means that a) the icon itself does _not_ resize & b) the label text has CSS applied that I don't want.

<img width="174" alt="Screenshot 2023-06-15 at 10 40 18 AM" src="https://github.com/AlaskaAirlines/auro-icon/assets/9616716/d62b2291-b86c-468e-8432-8b6ed8f21de6">

### Describe the solution you'd like

The ability to target the icon only while also using a label.

### Describe alternatives you've considered

When used `transform: scale`, which only works because we conveniently wanted the font-size to differ by the same proportion.

**Comment by blackfalcon** [comment]
@rallyb13 you are correct. The ability to resize never considered its association with the label content. 

@Patrick-Daly-AA This is a viable feature we should add to this element. @rallyb13 do you have availability to contribute to this update?

**Comment by Patrick-Daly-AA** [comment]
Assigning to @leeejune for input on if this is compliant with our design, or if the design needs to be updated to accommodate this.

**Comment by rallyb13** [comment]
`@rallyb13 do you have availability to contribute to this update?`
Not in the immediate future, I'm afraid. If it still hasn't been picked up after the hackathon, then maybe...?

**Comment by megrenn** [comment]
ADO ticket: https://dev.azure.com/itsals/E_Retain_Content/_workitems/edit/1375083
